originals/docker-terragrunt-fmt-master/.travis.yml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
Just a frozen state reference of original repositories. I can delete it if you like.

Reply:
I like the idea, and there might be an even better option (at least in my view):
Commit the original into master so that the 1st status is the one of the original at that time, and add some info about the original repo and commit hash to it. From there it's simple to diff, show -p and do interesting stuff in both repositories.
What do you think?

Reply:
We should've asked to for that original repository instead of creating new one.
What if in README I just add reference link to commit tree of original repositories?

Reply:
Yep a shallow clone would've been a good option.
We can go with the commit hashes only, I would remove the originals folder in any way. Can be looked up and will confuse maintainers in the long run.